code/src/java/pcgen/output/actor/FactKeyActor.java

```diff
@@ -40,6 +42,9 @@
  */
 public class FactKeyActor<T> implements OutputActor<CDOMObject>
 {
+	private static final ObjectWrapperFacet WRAPPER_FACET = FacetLibrary
```

Comment:
Will the FacetLibrary always be initialized before any reference to the FactKeyActor class is made?